src/main/java/edu/kis/powp/jobs2d/drivers/Jobs2dDriverTransformationDecorator.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class Jobs2dDriverTransformationDecorator implements Job2dDriver{
```

Comment:
The design should enable support for a wider range of transformations, e.g. movement, rotation, and enable their compositions.